examples/promptAPerson.yml

```diff
@@ -3,13 +3,30 @@ data-silos:
     description: Notify backend team to delete data
     integrationName: promptAPerson
     notify-email-address: test+2@transcend.io
```

Comment:
id bump the major version of the package and put this email address inside `prompt-a-vendor-email-settings`.  i can notify the customers that this affects, but it feels cleaner to nest them all together, especially after the PR that harris made last week to do the same grouping: https://github.com/transcend-io/main/pull/16504.